ActivityPub.Types/Util/JsonLDContext.cs

```diff
@@ -20,6 +18,9 @@ namespace ActivityPub.Types.Util;
 public class JsonLDContext
 {
     public HashSet<JsonLDContextObject> ContextObjects { get; }
+    
+    public JsonLDContext() {}
```

Comment:
We probably do need `[JsonConstructor]` here, sorry
```suggestion
    [JsonConstructor]
    public JsonLDContext() {}
```

Reply:
No worries.
I was working under the assumption that it was redundant due to using custom converter which inside of itself used the constructor with parameters

Reply:
You're probably right tbh, but its good to have this just in case. If we ever remove or replace the serializer, it would suddenly blow up.